src/routes/components/radio/+page.svelte

```diff
@@ -0,0 +1,130 @@
+	type Layout = 'vertical' | 'horizontal'
+
+	let radioCss: string = $state('Loading...')
+	let radioGroupCss: string = $state('Loading...')
+
+	let dimension: Dimension = $state('default')
+	let layout: Layout = $state('vertical')
+
+	// Svelte compiler breaks when it finds closing script tag, hence the need to make the template literal to have two parts
+	let useCode = $derived(
+		`<script lang="ts">
+    import RadioGroup from '$lib/components/ui/radio-button/radio-group.svelte'
+    import Radio from '$lib/components/ui/radio-button/radio.svelte'
+</script` +
+			`>
+
+<RadioGroup dimesion="${dimension}" layout="${layout}" name="rb">
+    {#snippet helperText()}
+        This is an optional helper text
+    {/snippet}
+    {#snippet label()}
+        Radio group label
+    {/snippet}
+    <Radio>Radio label</Radio>
+    <Radio>Radio label</Radio>
+</RadioGroup>
+`,
+	)
+
+	onMount(async () => {
+		const radio = await fetch('/generated/css/ui/radio-button/radio.css')
+		const radioGroup = await fetch('/generated/css/ui/radio-button/radio-group.css')
+		radioCss = await radio.text()
+		radioGroupCss = await radioGroup.text()
+	})
+</script>
+
+{#snippet description()}
+	Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quasi vitae explicabo, distinctio velit
+	nesciunt possimus dolorem exercitationem nobis aspernatur beatae animi culpa odit sequi eligendi
+	voluptate vel dolores, sint quos assumenda facilis? Nostrum, ipsum laudantium minus optio tempora
+	magnam dolorem suscipit vero explicabo, perferendis a voluptate aspernatur nesciunt. Sequi,
+	cumque!
+{/snippet}
+
+{#snippet examples()}{/snippet}
+
+{#snippet controls()}
+	<Select bind:value={dimension} label="Size">
+		<Option value="default">Default</Option>
+		<Option value="large">Large</Option>
+		<Option value="compact">Compact</Option>
+		<Option value="small">Small</Option>
+	</Select>
+	<Select bind:value={layout} label="Layout">
+		<Option value="vertical">Vertical</Option>
+		<Option value="horizontal">Horizontal</Option>
+	</Select>
+{/snippet}
+
+{#snippet helperText()}
+	This is an optional helper text
+{/snippet}
+{#snippet label()}
+	Radio group label
+{/snippet}
+
+{#snippet preview()}
+	<TabBar dimension="small">
+		<TabContent value="Preview">
+			<div class="preview-tabs preview-tab">
+				<RadioGroup {dimension} {layout} name="rb" {helperText} {label}>
+					<Radio>Radio label</Radio>
+					<Radio>Radio label</Radio>
+				</RadioGroup>
+			</div>
+		</TabContent>
+		<TabContent value="Svelte">
+			<Code language="svelte" code={useCode} class="preview-tab" />
+		</TabContent>
+	</TabBar>
+{/snippet}
+
+{#snippet implement()}
+	<TabBar dimension="small">
+		<TabContent value="Svelte"><Code language="svelte" code={radioGroup} /></TabContent>
+		<TabContent value="CSS"><Code language="css" code={radioGroupCss} /></TabContent>
+	</TabBar>
+	<TabBar dimension="small">
+		<TabContent value="Svelte"><Code language="svelte" code={radio} /></TabContent>
```

Comment:
The `radio-store.svelte` file is missing which is a dependency for both the `radio.svelte` and the `radio-group.svelte` files.